hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -442,7 +442,7 @@ public final class OzoneConfigKeys {
   public static final String OZONE_CLIENT_FAILOVER_MAX_ATTEMPTS_KEY =
       "ozone.client.failover.max.attempts";
   public static final int OZONE_CLIENT_FAILOVER_MAX_ATTEMPTS_DEFAULT =
-      500;
+      45;
```

Comment:
Instead of changing this for the client across the board, the CLI should change this. There might be other failover scenarios for long-running jobs where this high count is fine.

Reply:
Done.